charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.31.1
-version: 15.2.0
+version: 15.2.1
```

New feature should raise minor version 

```suggestion
version: 15.3.0
```

---

thanks! changed